### PR TITLE
OMID-231 Build and test Omid with Hadoop 3

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,7 @@ lib/
 *.iws
 *~
 *.swp
+^dev-support/artifacts
```
Comment (richardantal) [inline .gitignore:18]: I don't think we need the `^` character

Also I would add `dev-support/work` here too

File: pom.xml [hunk 2]
```diff
@@ -378,6 +378,8 @@
 
                         <!-- Exclude config and scripts -->
                         <exclude>**/dev-utils/*</exclude>
+                        <exclude>dev-support/artifacts/**</exclude>
```
Comment (richardantal) [inline pom.xml:381]: Also we should exclude this and   `dev-support/work/**` folders from `apache-rat-plugin` 
And maybe move those exclusions from a profile to apply globally but that could be handled as a separate task as it is not related to the scope of this change.

**Review by richardantal — COMMENTED** [review]
Looks good, left some minor comments

File: .gitignore (hunk 1 above)
Comment (stoty) [reply .gitignore:18]: Right, that is not a regex.

File: pom.xml (hunk 2 above)
Comment (stoty) [reply pom.xml:381]: Yes, that's not not strictly in scope, but makes testing the rat settings easier.

**Comment by stoty** [comment]
I hopefully fixed the problems you found, please check again.

**Review by richardantal — APPROVED** [review]
Thank you Istvan for the fix.
Now I have only the bintray-settings.xml found by rat check but I'll handle that as part of [OMID-189](https://issues.apache.org/jira/browse/OMID-189)

**Pull request closed without merging**